Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ScrollViewerExtensions/ScrollViewerExtensionsPage.xaml.cs

```diff
@@ -19,6 +19,14 @@ public sealed partial class ScrollViewerExtensionsPage : IXamlRenderListener
         public ScrollViewerExtensionsPage()
         {
             InitializeComponent();
+
```

Comment:
We have a lot of these patterns in the Sample app.  Can we add a section to the readme documentation about the 'Lifecycle of a Page' and have the guidelines for running code for the four scenarios: each time the sample loads (or unloads), and initialization/clean-up once for the lifetime of the app.

And would it make it clearer to just add them all to your ISampleNavigation interface and facilitate calling them all at the correct time?

Reply:
@WilliamABradley thoughts on this and comments on the change to the loading behavior?  I think that was my main question/concern.

Reply:
Sounds good.